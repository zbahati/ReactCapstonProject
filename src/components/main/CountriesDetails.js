import { Link, useParams } from 'react-router-dom';

const CountriesDetails = () => {
  const { detailsId } = useParams();
  return (
    <div>
      <h3>{detailsId}</h3>
      <Link to="/">Back to Home</Link>
    </div>
  );
};

export default CountriesDetails;
