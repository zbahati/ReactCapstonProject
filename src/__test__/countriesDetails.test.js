import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/store';
import CountriesDetails from '../components/main/CountriesDetails';

test('DetailsPage matches snapshot', () => {
  const { container } = render(
    <Provider store={store}>
      <Router>
        <CountriesDetails />
      </Router>
    </Provider>,
  );
  expect(container).toMatchSnapshot();
});
