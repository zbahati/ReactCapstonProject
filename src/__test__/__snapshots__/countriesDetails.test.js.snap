// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsPage matches snapshot 1`] = `
<div>
  <div>
    <div>
      Loading ..
    </div>
  </div>
</div>
`;
